src/main/java/io/pakland/mdas/githubstats/application/FetchTeamsFromOrganization.java

```diff
@@ -7,13 +7,13 @@
 import java.util.List;
 
 public class FetchTeamsFromOrganization {
-    private TeamExternalRepository teamRESTRepository;
+    private final TeamExternalRepository teamRESTRepository;
```

Comment:
Hmm, I forgot to rename this variables. Could you do it? Rename the `teamRESTRepository` for `teamExternalRepository`.